tasks/dataverse-postgres.yaml

```diff
@@ -1,40 +1,36 @@
+---
+# dataverse/tasks/dataverse-postgres.yaml
+
 - name: install postgres on RedHat / CentOS 7.
   yum: name=postgresql-server state=latest
   when: ansible_os_family == "RedHat" and
         ansible_distribution_major_version == "7"
-  tags: postgres
 
 - name: init postgres on RedHat / CentOS 7.
-  shell: /usr/bin/postgresql-setup initdb
+  shell: creates=/var/lib/pgsql/data /usr/bin/postgresql-setup initdb
```

Comment:
The addition of `creates=…` allows the task to be run twice. Previously, only the first run of the playbook could succeed; afterwards it would fail on this task. In general, we should strive to write idempotent Ansible tasks.